### PR TITLE
Update README with more informations and fewer WARNINGS

### PR DESCRIPTION
**Pull request by bricesanchez**
[ci skip]

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,27 +1,55 @@
...
+## Install
 
-Ensure you have created your application's database before adding this engine (with `rake db:setup`).
-
-Open your `Gemfile` and add this line to the bottom:
+Open up your ``Gemfile`` and add at the bottom this line:
 
 ```ruby
-gem 'refinerycms-portfolio', github: 'refinery/refinerycms-portfolio', branch: 'master'
+gem 'refinerycms-portfolio', git: 'https://github.com/refinery/refinerycms-portfolio', branch: 'master'
```
Comment (simi) [inline readme.md]: Why?

File: readme.md [hunk 2]
```diff
@@ -1,27 +1,55 @@
...
 ```
 
-Now run `bundle install` and once bundler has installed the gem run:
+Now, run ``bundle install``
```
Comment (simi) [inline readme.md]: Why double ticks?

**Comment by simi** [comment]
It will be nice to add generated binstubs to repo and use them in README since some commands are noted with/without `bundle exec`.

File: readme.md (hunk 1 above)
Comment (bricesanchez) [inline readme.md]: i fix that (copy/paste from refinerycms-blog readme)

File: readme.md (hunk 2 above)
Comment (bricesanchez) [inline readme.md]: Same problem

**Comment by bricesanchez** [comment]
@simi Is it ok for you?

**Comment by simi** [comment]
Good work @bricesanchez. One note - it will be nice to use bundled rake in README.

**Comment by bricesanchez** [comment]
:up: :)

**Comment by bricesanchez** [comment]
:wave: @parndt This PR has less scary informations too

**Pull request merged**